Replace sot_red_alert, sot_yellow_alert, sot_safemode_alert @cfa lists with @ipa lists

malgosias opened the pull request:
Replace sot_red_alert, sot_yellow_alert, sot_safemode_alert @cfa.harvard.edu mailing lists with @ipa.harvard.edu mailing lists. Files affected: ``check_state.pm``.

If approved, then the ''check_state.pm'' file will be copied to the primary alert location at ``/data/mta4/www/Snapshot/``.

In case of issues, this operation will be reverted by copying the preceding version of the ``check_state.pm`` file (base branch hrc-bside-swap-step1) to ``/data/mta4/www/Snapshot/``.

malgosias commented:
This PR replaces google mailing lists with the IPA mailing lists, following the failure of google mailings list to deliver to the SMS addresses. Since currently there is no way to test these updates other than a failure in real time data, I requested a review by three reviewers.

linamaria11 reviewed (approved):
confirmed with original file that there are no more changes than these to make, all changes look good
is the mycingular number on line 1419 still valid?

malgosias commented:
Updated mailing lists and messages in ``check_state_alerts.pm`` (backup) to ``match check_state.pm`` (primary):
- Corrected scs107, fmt, brit and nsun messages/subjects to match ``check_state.pm``
- Replaced ...alert@cfa with ...alert@ipa
- A few changes in commented lines to match check_state.pm for easier comparison

Expected differences between ``check_state_alerts.pm`` (backup) and ``check_state.pm`` (primary):
- In the locations of the .*wait, .*alert, *par files
- In the definition of comfile (for dsn_summary.dat)

Note that ``check_state_noalerts.pm`` has not been maintained for messages/mailing lists consistency with the check_state.pm because the function of this file is simply to block alerting capabilities of the backup without interfering with the general flow of the primary/backup code. This is achieved by commenting all the ``send_xxx_alert`` calls in ``check_state_noalerts.pm``.
